Soccar [UI]/src/nl/soccar/ui/fx/controller/SessionViewFXMLController.java

```diff
@@ -121,12 +151,45 @@ private void leaveRoom() {
     }
 
     /**
-     * Method that navigates to the game view and set the application window to
-     * full screen mode.
+     * Navigates to the game view and set the application window to full screen
+     * mode.
      */
-    public void startGame() {
+    private void startGame() {
         Connection connection = ClientController.getInstance().getCurrentConnection();
         
         connection.send(new StartGameMessage());
     }
+
+    /**
+     * Sends the chatmessage to all players in this session.
+     */
+    private void sendChatMessage() {
+        String message = txtChat.getText();
+        if (message.trim().isEmpty() || message.length() >= 75) {
+            return;
+        }
+        
+        ClientController.getInstance().getCurrentConnection().send(new ChatMessage(message));
+        
+        txtChat.clear();
+    }
+    
+    public void addChatMessage(String username, Privilege privilege, String message) {

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)